flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveDialectAggITCase.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.connectors.hive;
+
+import org.apache.flink.types.Row;
+import org.apache.flink.util.CollectionUtil;
+
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.apache.flink.table.planner.utils.TableTestUtil.readFromResource;
+import static org.assertj.core.api.Assertions.assertThat;
+
+/** Test for native hive agg function compatibility. */
+public class HiveDialectAggITCase extends HiveDialectITCaseBase {
```

Comment:
I just want to make the test class function more clear, because there will be more tests in the future as we implement other agg functions. But I remove this test class now, we can separate it if needed in the future.

Reply:
I mean we can just create `HiveDialectAggITCase` without extend `HiveDialectITCaseBase`, and then put these test to `HiveDialectAggITCase`. I think it'll be better to move these tests to `HiveDialectAggITCase` in the following pr to make avoid HiveDialectQueryiTCase contains much things.